web/src/overlays_modals.ts

```diff
@@ -0,0 +1,5 @@
+import * as overlays from './overlays';
+
+export function any_active(): boolean {
+    return overlays.is_active() || overlays.is_modal_open();
+}
```

Comment:
I think it would probably be cleaer to just have each caller inline this, given there aren't a ton?

Reply:
We can discuss in #frontend and change in a follow-up if desired.

Reply:
I will open a frontend discussion.` if (overlays.is_active() || modals.is_open()) {`, whould probably rename overlays.is_active to is_open before doing this.

Reply:
https://chat.zulip.org/#narrow/stream/6-frontend/topic/overlays_modals.20.2327140